modules/gis/test/data/empty_binary.js

```diff
@@ -0,0 +1,26 @@
+export default {
+  points: {
+    positions: {value: {}, size: null},
+    globalFeatureIds: {value: {}, size: 1},
+    featureIds: {value: {}, size: 1},
+    numericProps: {},
+    properties: []
+  },
+  lines: {
+    pathIndices: {value: {0: 0}, size: 1},
```

Comment:
I'd suggest a TypedArray of length 1 instead. The current `{0: 0}` relies on the fact that the same API works for numbered objects and arrays.